src/js/components/CommandPanel/CommandPanel.tsx

```diff
@@ -226,11 +227,13 @@ const CommandPanel = ({setLogs}: {setLogs: TSetLogs}) => {
             return;
         }
 
+        const bufferWithoutComments = removeComments(buffer);
```

Comment:
seems `buffer` in this context can be replaced with `hex` or `hexDump`

Reply:
replaced `buffer` with `hex`